Apps/Common/src/Auditing/AuditLogger.cs

```diff
@@ -57,7 +57,7 @@ public void PopulateWithHttpContext(HttpContext context, AuditEvent auditEvent)
             auditEvent.ApplicationType = GetApplicationType();
             auditEvent.TransactionResultCode = GetTransactionResultType(context.Response.StatusCode);
 
-            auditEvent.ApplicationSubject = hdid ?? subjectPhn ?? subjectQuery;
+            auditEvent.ApplicationSubject = subjectQuery ?? hdid ?? subjectPhn;
```

Comment:
As Mike indicated logic needs to be made smarter for the APIs (not for Admin) and should check route values and query parms for HDID.

Reply:
Route values and query parameters are now checked for Hdid.